### PR TITLE
feat(card): typescript

### PR DESCRIPTION
**Pull request by moog16**
related #48

### CODE REVIEW

File: packages/card/PrimaryContent.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  initRipple,
+  children,
+  unbounded, // eslint-disable-line no-unused-vars
+  /* eslint-enable react/prop-types */
+  ...otherProps
+}) => {
+  const classes = classnames('mdc-card__primary-action', className);
+
+  return (
+    <div className={classes} ref={initRipple} {...otherProps}>
```
Comment (moog16) [inline packages/card/PrimaryContent.tsx]: if new line props indent 4 spaces

File: packages/card/ActionButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+import * as React from 'react';
+import * as classnames from 'classnames';
+
+type ChildType = React.ReactElement<React.HTMLProps<HTMLButtonElement|HTMLAnchorElement>>;
```
Comment (moog16) [inline packages/card/ActionButtons.tsx]: maxlen 80!

File: packages/card/ActionButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+import * as React from 'react';
```
Comment (williamernest) [inline packages/card/ActionButtons.tsx]: Add new line.

File: packages/card/ActionIcons.tsx [hunk 4]
```diff
@@ -19,46 +19,38 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
+import * as React from 'react';
```
Comment (williamernest) [inline packages/card/ActionIcons.tsx]: Add new line

File: packages/card/Actions.tsx [hunk 5]
```diff
@@ -19,42 +19,25 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
+import * as React from 'react';
```
Comment (williamernest) [inline packages/card/Actions.tsx]: add new line

File: packages/card/Media.tsx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+import * as React from 'react';
```
Comment (williamernest) [inline packages/card/Media.tsx]: new line

File: packages/card/PrimaryContent.tsx [hunk 7]
```diff
@@ -19,46 +19,33 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
-import React from 'react';
-import classnames from 'classnames';
-import PropTypes from 'prop-types';
-
+import * as React from 'react';
```
Comment (williamernest) [inline packages/card/PrimaryContent.tsx]: add new line

File: packages/card/PrimaryContent.tsx [hunk 8]
```diff
@@ -19,46 +19,33 @@
...
+import * as classnames from 'classnames';
+// @ts-ignore
```
Comment (williamernest) [inline packages/card/PrimaryContent.tsx]: Add comment for why this is needed.

File: packages/card/index.tsx [hunk 9]
```diff
@@ -19,56 +19,38 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
-import React from 'react';
-import classnames from 'classnames';
-import PropTypes from 'prop-types';
+import * as React from 'react';
```
Comment (williamernest) [inline packages/card/index.tsx]: add new line

**Pull request merged**